backend/db/migrate/20220812075824_add_impact_calculated_to_projects.rb

```diff
@@ -0,0 +1,6 @@
+class AddImpactCalculatedToProjects < ActiveRecord::Migration[7.0]
+  def change
+    add_column :projects, :impact_calculated, :boolean, default: false
+    Project.update_all impact_calculated: true # we expect that all existing projects already have impact calculated
```

Comment:
I was about to write a comment to make this reversible, but I see that rollback works just fine with this lol.